Parsers/ASimUserManagement/Parsers/vimUserManagementSentinelOne.yaml

```diff
@@ -0,0 +1,185 @@
+      | extend GroupName = iff(GroupName == "null", "", GroupName)
+      | project-rename
+          EventStartTime = createdAt_t,
+          SrcIpAddr = ipAddress,
+          EventUid = _ItemId,
+          ActorUserId = id,
+          GroupId = groupId_s,
+          EventMessage = primaryDescription_s,
+          EventOriginalUid = activityUuid_g
+      | extend
+          EventCount = int(1),
+          EventResult = "Success",
+          DvcAction = "Allowed",
+          EventSeverity = "Informational",
+          EventSchema = "UserManagement",
+          EventSchemaVersion = "0.1.1",
+          EventProduct = "SentinelOne",
+          EventVendor = "SentinelOne",
+          EventResultDetails = "Other"
+      | extend
+          Dvc = EventProduct,
+          EventEndTime = EventStartTime,
+          IpAddr = SrcIpAddr,
+          User = ActorUsername,
+          UpdatedPropertyName = EventSubType,
+          ActorUserIdType = iff(isnotempty(ActorUserId),"Other",""),
+          ActorUserType = _ASIM_GetUserType(ActorUsername,ActorUserId),
+          ActorUsernameType = _ASIM_GetUsernameType(ActorUsername),
+          GroupIdType = iff(isnotempty(GroupId), "UID", ""),
+          GroupNameType = iff(isnotempty(GroupName), "Simple", ""),
+          GroupType = iff(isnotempty(groupType), "Other", ""),
+          GroupOriginalType = groupType,
+          TargetUsernameType = _ASIM_GetUsernameType(TargetUsername),
+          TargetUserType = _ASIM_GetUserType(TargetUsername, ""),
+          AdditionalFields = bag_pack(
+                        "userScope", userScope,
+                        "scopeLevelName", scopeLevelName,
+                        "scopeName", scopeName,
+                        "modifiedFields", modifiedFields,
+                        "roleName", roleName,
+                        "deactivationPeriodInDays", deactivationPeriodInDays,
+                        "descriptionChanged", descriptionChanged
+                    )
+      | project-away 
+          *_b,
+          *_d,
+          *_g,
+          *_s,
+          *_t,
+          byUser,
+          username,
+          email,
+          group,
+          groupName,
+          groupType,
+          name,
+          oldDescription,
+          oldRole,
+          description,
+          role,
+          userScope,
+          scopeLevelName,
+          scopeName,
+          roleName,
+          modifiedFields,
+          ModifiedFields,
+          deactivationPeriodInDays,
+          descriptionChanged,
+          restOfMessage,
+          _ResourceId,
+          TenantId,
+          RawData,
+          Computer,
+          MG,
+          ManagementGroupName,
+          SourceSystem
+  };
+  parser(
+    starttime = starttime,
+    endtime = endtime,
+    targetusername_has = targetusername_has,
+    actorusername_has = actorusername_has,
+    targetdomain_has = targetdomain_has,
+    anydomain_has = anydomain_has,
+    disabled=disabled
+  )
```

Comment:
Team, no threat fields found for user management?

Reply:
No threat field available for user related data in sentinel.